src/js/view/controls/components/simple-tooltip.js

```diff
@@ -39,6 +39,7 @@ export function SimpleTooltip(attachToElement, name, text, openCallback) {
     };
 
     attachToElement.addEventListener('mouseover', instance.open);
+    attachToElement.addEventListener('focus', instance.open);
     attachToElement.addEventListener('mouseout', instance.close);
     attachToElement.addEventListener('touchstart', (evt) => {
         instance.touchEvent = getPointerType(evt) === 'touch';
```

Comment:
On an unrelated note, this use of `getPointerType` is completely unnecessary. This statement will always return `true`, because this is a 'touchstart' (TouchEvent) listener and `getPointerType` will always return `true` for TouchEvents. @hghazzi added this code.